src/styles.scss

```
@@ -144,6 +173,28 @@ mat-card,
   color: var(--text-primary) !important;
 }
 
+/* Ensure disabled fields have enough contrast in dark theme */
+.mdc-text-field--disabled {
+  opacity: 1 !important; /* Material default lowers opacity of the whole container */
+  background-color: var(--mdc-outlined-text-field-disabled-container-color) !important;
+  border-radius: 4px; /* Optional: adds a slight rounding to the background fill */
+
+  .mdc-text-field__input {
+
+    color: var(--mdc-outlined-text-field-disabled-input-text-color) !important;
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)